### DISCUSSION TITLE
Composed time block

### DISCUSSION THREAD

**Discussion by dokato** [jspsych/jsPsych#684] [Q&A] [answered]
Is it possible to create time block composed of two stimuli's? In particular I want to have random dots, which I can easily achieve using RDK module: https://www.jspsych.org/plugins/jspsych-rdk/. But now I want to display also an image beneath the moving dots at the same time. What's the easiest way to achieve that?

**Comment by jodeleeuw** [comment, marked as answer]
Unfortunately you can't combine two plugins at the same time. 

The easiest way to do what you want may be to create a copy of the RDK plugin and add a small modification to display the image below the stimulus. You could reference the code in `image-keyboard-response` as a template for how to render the image. 

If you run into trouble with that you can post here for guidance!

**Comment by dokato** [comment]
Thanks for a quick reply.

> Unfortunately you can't combine two plugins at the same time.

Yeah, that's what I figured and indeed, modifying js code for RDK was the easiest option. I think it would be a nice feature to have, though...
